docs/roles-and-permissions.md

```diff
@@ -0,0 +1,31 @@
+# Roles and Permissions 
+
+> WordPress has six pre-defined roles: Super Admin, Administrator, Editor, Author, Contributor and Subscriber. Each role is allowed to perform a set of tasks called Capabilities.
+
+These roles and their permissions can be explored in detail here: [https://wordpress.org/support/article/roles-and-capabilities/](https://wordpress.org/support/article/roles-and-capabilities/)
+
+Inside of the web stories app we are harnessing these role definitions to manage what users have permission to do. This document is meant to be a reference of what these roles mean for web stories specifically. 
+
+| Has permission? | Role |
+| --- | --- |
+| Can create new story | Admin, Editor, Author |
+| Can duplicate story XYZ | Admin |
+| Can publish story | Admin |
+| Can edit story | Admin, Editor |
+| Can delete any story | Admin |
+| Can delete their own story | Admin, Editor, Author |
+| Can upload media | Admin |
+| Can upload or delete publisher logos | Admin |
+| Can edit settings | Admin|
+| Can update video optimization settings | Admin |
+
+
+In cases where a permission is not granted, we need to make sure the UI reflects this.  
+
+
+## Web Stories specific capabilities 
+
+
+`hasUploadMediaAction` : admin 
+`hasPublishAction`: admin, editor if story is their own 
+`hasAssignAuthorAction`: admin
```

Comment:
These are not really Web Stories specific in any way.

`hasUploadMediaAction` is just a fancy name for the `upload_files` capability in WordPress. Which by default admins, editors, and authors have.

`hasPublishAction`: fancy name for `publish_posts`. admins, editors, and authors have that by default.

`hasAssignAuthorAction`: fancy name for `edit_others_posts`, which by default only admin and editor have

Reply:
Sure, i get that, but most of us on the project aren't touching wordPress too much and having some baseline for how the capabilities users have in the app tie to wordPress could help alleviate the disconnect that Jonny brought up with not taking care of users with less permissions off the bat.

Reply:
For that I think it would be preferable to have a section here outlining how to check for permissions, with some examples including some of these here.

Reply:
ok!